selectbox.lua

```diff
@@ -26,6 +26,7 @@ function SelectBox:new(parent, label)
   self.size.x = 200 + (style.padding.x * 2)
   self.size.y = self:get_font():get_height() + (style.padding.y * 2)
   self.list_container = Widget()
+  self.list_container.name = self.name
```

Comment:
A reminder that with the approach mentioned in #2 of using get_name to iterate the parents at runtime this would be:

```suggestion
  self.list_container.name = self:get_name()
```